assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -43,6 +43,26 @@ function createXScale(values, xSize) {
         .domain(xExtent);
 }
 
+/**
+ * Create the scale based on the parameter code
+ *
+ * @param parmCd
+ * @param extent
+ * @param size
+ */
+function yScaleByParameter(parmCd, extent, size) {
+    const symlogParams = ['00060'];
```

Comment:
Should this be a module level constant? I would prefer it use all caps at the very least.

Reply:
Sure. It can be a module level constant.